com.epam.zoo/src/Crocodile.java

```diff
@@ -1,15 +1,15 @@
-public class Crocodile extends Animal {
-    public Crocodile(String name, ZooType zooType) {
-        super(name, "meat", 18);
-    }
-
-    @Override
-    public String getEnclosure() {
-        return "Crocodile Pool";
-    }
-
-    @Override
-    public String getCageType() {
-        return "";
-    }
-}
+public class Crocodile extends Animal {
+    public Crocodile(String name, ZooType zooType) {
+        super(name, "meat", 18);
```

Comment:
Remove all inline strings, replace them with:
private static final String MEAT = "meat";

And then use them in methods